metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/usage.py

```diff
@@ -150,15 +150,10 @@ class BigQueryUsageExtractor:
     * Aggregation of these statistics into buckets, by day or hour granularity
 
     :::note
-    1. This source only does usage statistics. To get the tables, views, and schemas in your BigQuery project, use the `bigquery` plugin.
-    2. Depending on the compliance policies setup for the bigquery instance, sometimes logging.read permission is not sufficient. In that case, use either admin or private log viewer permission.
+    1. Depending on the compliance policies setup for the bigquery instance, sometimes logging.read permission is not sufficient. In that case, use either admin or private log viewer permission.
     :::
     """
 
-    aggregated_info: Dict[
```

Comment:
out of curiousity, what was the reason for moving this into `generate_usage_for_project`?

Reply:
Because I wanted to make sure don't keep in memory this dict longer than we should and making sure if we run the extract usage multiple times then it won't add others project usage to the same dict.

Reply:
Makes sense